ersilia/cli/commands/catalog.py

```diff
@@ -53,7 +53,7 @@ def catalog(local=False, file_name=None, browser=False, more=False):
             )
             return
         if file_name is None:
-            catalog = catalog_table.as_json()
+            catalog = catalog_table.as_table()
```

Comment:
I don't think we should change this from `as_json` to `as_table`. We should instead have a flag called `--as-table` and check if it is set, and only then return the result as a table.

Reply:
@DhanshreeA Thanks for the feedback,  and I agree , the json output as the default, and the table format output when specified makes a lot more sense.
I will implement the flag so the output can be conditionally formatted as a table when specified.

Reply:
@DhanshreeA I have implemented the --as-table flag in the latest commits. Now the default behaviour remains the same, returning the json output, unless the --as-table flag is added to the command.